keps/sig-api-machinery/2340-Consistent-reads-from-cache/README.md

```diff
@@ -320,95 +382,50 @@ needed.
-<!--
-Describe manual testing that was done and the outcomes.
-Longer term, we may want to require automated upgrade/rollback tests, but we
-are missing a bunch of machinery and tooling and can't do that now.
--->
+No need for tests as this feature only change apiserver behavior without any perisstent side effects.
 
 ###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
 
-<!--
-Even if applying deprecation policies, they may still surprise some users.
--->
+We will want to deprecate support of etcd v3.3 in Kubernetes 1.29 and drop the support in v1.32.
+etcd v3.3 is almost 6 years old release, is no longer officially supported with the last update being 2 years ago.
 
 ### Monitoring Requirements
 
-<!--
-This section must be completed when targeting beta to a release.
-
-For GA, this section is required: approvers should be able to confirm the
-previous answers based on experience in the field.
--->
-
 ###### How can an operator determine if the feature is in use by workloads?
 
-<!--
-Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
-checking if there are objects with field X set) may be a last resort. Avoid
-logs or events for this purpose.
--->
+No workload visible change. 
 
 ###### How can someone using this feature know that it is working for their instance?
 
-<!--
-For instance, if this is a pod-related feature, it should be possible to determine if the feature is functioning properly
-for each individual pod.
-Pick one more of these and delete the rest.
-Please describe all items visible to end users below with sufficient detail so that they can verify correct enablement
-and operation of this feature.
-Recall that end users cannot usually observe component logs or access metrics.
--->
-
-- [ ] Events
-  - Event Reason:
-- [ ] API .status
-  - Condition name:
-  - Other field:
-- [ ] Other (treat as last resort)
-  - Details:
+Check if 99th percentile of `apiserver_watch_cache_read_wait` metric is below 200ms.
 
 ###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
 
-Use existing kube-apiserver SLOs. 
-
-TODO: Provide link
+https://github.com/kubernetes/community/blob/master/sig-scalability/slos/api_call_latency.md
 
 ###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
 
-- [ ] Metrics
-  - Metric name: TODO: provide exact name of apiserver latency metric
-  - [Optional] Aggregation method:
-  - Components exposing the metric:
+https://github.com/kubernetes/community/blob/master/sig-scalability/slos/api_call_latency.md
 
 ###### Are there any missing metrics that would be useful to have to improve observability of this feature?
 
-Watch latency metric.
+Watch latency metric, however it is not known how to measure it correctly.
+Proposed metric `apiserver_watch_cache_read_wait` should be a good enough approximation of healthiness of this feature.
```

Comment:
L434 - it depends on the etcd being in high enough version - can you please add it?

Reply:
Done